manim/logger.py

```diff
@@ -4,17 +4,23 @@
 from rich.console import Console
 from rich.logging import RichHandler
 from rich.theme import Theme
+from rich import print as printf
+from rich import errors,color
 
 from .utils.config_utils import config_parser, successfully_read_files
 
 
 def parse_theme(fp):
     config_parser.read(fp)
     theme = dict(config_parser["log.color"])
-    customTheme = Theme(theme)
+    try:
+        customTheme = Theme(theme)
+    except (color.ColorParseError,errors.StyleSyntaxError):
+        customTheme = None
+        printf("[logging.level.error]Looks like your colour configuration has an error. So loading default color configuration[/logging.level.error]")
+        #console.log(e) #Not logging it as it looks clumsy
```

Comment:
Please delete this line. In general, do not leave commented out lines in the codebase :)